Disable optional questions for input during installation of go-db-perl
author: Alex Domingo (Vrije Universiteit Brussel)
--- Makefile.PL.orig	2024-12-21 00:41:02.514044000 +0100
+++ Makefile.PL	2024-12-21 00:42:23.509639000 +0100
@@ -173,28 +173,28 @@
 
 my $default = $admin->dbhost || "localhost";
 print "\nmysql server address [$default]:";
-my $svr = <STDIN>;
+my $svr = "";
 chomp $svr;
 if (!$svr) {$svr = $default}
 $admin->dbhost($svr);
 
 $default = $admin->dbname || "go";
 print "\nname of readonly populated GO database[$default]:";
-my $db = <STDIN>;
+my $db = "";
 chomp $db;
 if (!$db) {$db = $default}
 $admin->dbname($db);
 
 $default = $admin->dbuser || "";
 print "\nname of dbuser (optional)[$default]:";
-my $dbuser = <STDIN>;
+my $dbuser = "";
 chomp $dbuser;
 if (!$dbuser) {$dbuser = $default}
 $admin->dbuser($dbuser);
 
 $default = $admin->dbauth || "";
 print "\ndb password (optional)[$default]:";
-my $dbauth = <STDIN>;
+my $dbauth = "";
 chomp $dbauth;
 if (!$dbauth) {$dbauth = $default}
 $admin->dbauth($dbauth);
